hal/halx86/acpi/madt.c

```diff
@@ -127,23 +90,23 @@ HalpParseApicTables(
 
                 if (AcpiHeader->Length != sizeof(*LocalApic))
                 {
-                    DPRINT01("Type/Length mismatch: %p, %u\n", AcpiHeader, AcpiHeader->Length);
+                    DPRINT("Type/Length mismatch: %p, %u\n", AcpiHeader, AcpiHeader->Length);
                     return;
                 }
 
-                DPRINT00(" Local Apic, Processor %lu: ProcessorId %u, Id %u, LapicFlags %08X\n",
+                DPRINT(" Local Apic, Processor %lu: ProcessorId %u, Id %u, LapicFlags %08X\n",
                          HalpApicInfoTable.ProcessorCount,
                          LocalApic->ProcessorId, LocalApic->Id, LocalApic->LapicFlags);
 
                 if (!(LocalApic->LapicFlags & (LAPIC_FLAG_ONLINE_CAPABLE | LAPIC_FLAG_ENABLED)))
                 {
-                    DPRINT00("  Ignored: unusable\n");
+                    DPRINT("  Ignored: unusable\n");
                     break;
                 }
 
                 if (HalpApicInfoTable.ProcessorCount == _countof(HalpStaticProcessorIdentity))
                 {
-                    DPRINT00("  Skipped: array is full\n");
+                    DPRINT("  Skipped: array is full\n");
```

Comment:
What's the point of these DPRINT's when there is no debugger to display them anyway?

Reply:
They display now